subprojects/language-java/src/main/java/org/gradle/api/tasks/javadoc/Javadoc.java

```diff
@@ -94,7 +96,7 @@
 
     private String maxMemory;
 
-    private MinimalJavadocOptions options = new StandardJavadocDocletOptions();
+    private StandardJavadocDocletOptions options = new StandardJavadocDocletOptions();
```

Comment:
We need this so that the annotations declared in `StandardJavadocDocletOptions` are picked up. It's a weird convention, but if there's a field with the property name derived from the getter, then `@Nested` uses the type of the field instead of the return type of the getter.